src/vs/platform/prompts/common/config.ts

```diff
@@ -132,7 +132,7 @@ export namespace PromptsConfig {
  * 			be clearly mapped to a boolean (e.g., `"true"`, `"TRUE"`, `"FaLSe"`, etc.),
  * 			`undefined` for rest of the values
  */
-function asBoolean(value: any): boolean | undefined {
+export const asBoolean = (value: any): boolean | undefined => {
```

Comment:
@legomushroom is there any advantage to changing this to use a constant + arrow syntax versus the function keyword?

Reply:
Mainly for consistency - I usually use the lambda function definition everywhere else, and this one was left over for some reason.
Not required for this PR of course, but these things tend to split through the cracks if you don't update when you see them 🤷